tests/integration/kubernetes/confidential_kbs.sh

```diff
@@ -322,6 +323,16 @@ function kbs_k8s_deploy() {
 			# ITA/ITTS specific configuration
 			sed -i -e "s/tBfd5kKX2x9ahbodKV1.../${ITA_KEY}/g" kbs-config.toml
 		popd
+		
+		# Ideally this should be something kustomizable on trustee side.
+		#
+		# However, for now let's take the bullet and do it here, and revert this as
+		# soon as https://github.com/confidential-containers/trustee/issues/567 is
+		# solved.
+		pushd "${COCO_KBS_DIR}/config/kubernetes/base/"
```

Comment:
Is if worth having a `[ -n "${HTTPS_PROXY}" ]` check to avoid do this is not required?

Reply:
It is worth the ${HTTS_PROXY} check, indeed.